x-pack/plugins/secops/public/components/link_to/redirect_to_network.tsx

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+
+export const RedirectToNetworkPage = () => {
+  return <Redirect to={`/network`} />;
+};
+
+export const getNetworkUrl = () => {
+  return `#/link-to/network`;
+};
```

Comment:
Similar to above with Hosts with comments about interpolation.

Reply:
same thing It will change soon enough when I add the KQL